ui/vty-lib/src/Ariadne/UI/Vty/Widget/Wallet.hs

```diff
@@ -48,6 +48,12 @@ data RenameResult
   | RenameResultError !Text
   | RenameResultSuccess
 
+data BalanceResult
```

Comment:
Why is it copy-pasted? Isn't there a better solution?

Reply:
I consider this widget-local structure.
Other way would be defining polymorphic result type `Result a` and using it for all requests (as they only differ by result type), but that's for another PR.